cli/lib/cli.js

```diff
@@ -25,6 +26,10 @@ const coerceFalse = (arg) => {
   return arg !== 'false'
 }
 
+const coerceInt = (arg) => {
+  return typeof arg === 'string' ? parseInt(arg) : arg
+}
```

Comment:
this function is a misnomer, it doesn't always coerce to int, nor does it coerce only ints

so, is `coerceInt({}) === {}` the desired behavior here? i'd expect a fn named `coerceInt` to return an integer

Reply:
I don't want to change the existing logic, and I agree this function should be named `corceStringToInt`

Reply:
renamed